src/System.Management.Automation/engine/Utils.cs

```diff
@@ -798,21 +798,119 @@ private static T GetPolicySettingFromConfigFile<T>(ConfigScope[] preferenceOrder
 #if !UNIX
         private static readonly Dictionary<string, string> GroupPolicyKeys = new Dictionary<string, string>
         {
-            {nameof(ScriptExecution), @"Software\Policies\Microsoft\PowerShellCore"},
-            {nameof(ScriptBlockLogging), @"Software\Policies\Microsoft\PowerShellCore\ScriptBlockLogging"},
-            {nameof(ModuleLogging), @"Software\Policies\Microsoft\PowerShellCore\ModuleLogging"},
+            {nameof(ScriptExecution), @"Software\Policies\PowerShellCore"},
+            {nameof(ScriptBlockLogging), @"Software\Policies\PowerShellCore\ScriptBlockLogging"},
+            {nameof(ModuleLogging), @"Software\Policies\PowerShellCore\ModuleLogging"},
```

Comment:
Discussed offline.  The intent in the RFC was that there would not be a change to the path.  We should revise the RFC.

Reply:
Opened https://github.com/PowerShell/PowerShell-RFC/pull/224

Reply:
Updated code according to an update to RFC. :)